packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -91,3 +95,15 @@ function MapsIndoorsMap(props) {
 }
 
 export default MapsIndoorsMap;
+
+
+/**
+ * Convert query parameter value (which is always a string) into a boolean.
+ * It will only accept the string 'true' as a boolean true. Anything else will return false.
+ *
+ * @param {string} queryParameterValue
+ * @return {boolean}
+ */
+function getBooleanQueryParameter(queryParameterValue) {
+    return queryParameterValue === 'true' ? true : false;
```

Comment:
it can even be shortened to

```suggestion
    return queryParameterValue === 'true';
```